packages/manager/src/config/feedbackReq.ts

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+import { env } from './env';
+
+export const feedbackReq = axios.create({
+  baseURL: env.PUBLIC_FEEDBACK_BASE_URL,
+  headers: {
+    'Content-Type': 'application/json',
+    Accept: 'application/json'
+  }
+});
+
+feedbackReq.interceptors.request.use((config: any) => {
```

Comment:
Please declare the type for the `config`. Remove `any` from here.

Reply:
fixed